statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -734,14 +1001,24 @@ def forecast_interval(self, y, steps, alpha=0.05):
         assert(0 < alpha < 1)
         q = util.norm_signif_level(alpha)
 
-        point_forecast = self.forecast(y, steps)
+        point_forecast = self.forecast(y, steps, exog_future=exog_future)
         sigma = np.sqrt(self._forecast_vars(steps))
 
         forc_lower = point_forecast - q * sigma
         forc_upper = point_forecast + q * sigma
 
         return point_forecast, forc_lower, forc_upper
```

Comment:
Happy to help.  I'm looking forward to playing around with all this new functionality.

If you fix the docstring issue, you can add a note that it closes issue 3615.